app/build.gradle

```diff
@@ -244,10 +246,11 @@ dependencies {
     androidTestCompile 'com.android.support.test.espresso:espresso-core:2.2.2'
     androidTestCompile 'com.android.support.test.espresso:espresso-intents:2.2.2'
     testCompile 'junit:junit:4.12'
-    testCompile "org.jetbrains.kotlin:kotlin-stdlib:1.0.6"
     testCompile "org.jetbrains.kotlin:kotlin-test-junit:1.0.6"
     testCompile 'com.taroid.knit:knit:0.1.2'
-    testCompile 'com.github.sys1yagi:kmockito:0.1.2'
+    testCompile ('com.github.sys1yagi:kmockito:0.1.2'){
+        exclude group: 'org.jetbrains.kotlin', module: 'kotlin-stdlib'
```

Comment:
Could you please use `resolutionStrategy` to specify versions definitely?
It's a redundant task to add exclude-clause whenever we introduce any kotlin libraries.